lib/test/tester.js

```diff
@@ -222,6 +222,44 @@ class Tester {
     this.coverageFile = path.join(this.tempDir, 'coverage.out')
   }
 
+  buildGoTestArgs (timeoutMillis = 60000) {
+    const args = ['test', '-coverprofile=' + this.coverageFile]
+    const configFlags = atom.config.get('go-plus.test.additionalTestFlags')
+
+    let shortFlag = false
+    let verboseFlag = false
+
+    if (configFlags && configFlags.length) {
+      const arr = configFlags.trim().split(/\s+/g)
```

Comment:
parsing args this way is not correct, e.g.: `-my-arg="hello world"`

the [yargs-parser](https://github.com/yargs/yargs-parser) seems to parse them correctly